NitroxServer/ConsoleCommands/WarpCommand.cs

```diff
@@ -1,25 +1,24 @@
 ﻿using NitroxModel.DataStructures.GameLogic;
-using NitroxModel.Helper;
 using NitroxServer.ConsoleCommands.Abstract;
 using NitroxServer.ConsoleCommands.Abstract.Type;
 
 namespace NitroxServer.ConsoleCommands
 {
     internal class WarpCommand : Command
     {
-        public WarpCommand() : base("warpto", Perms.ADMIN, "Teleports you on a player")
+        public WarpCommand() : base("warp", Perms.ADMIN, "Allows to teleport players")
         {
             AddParameter(new TypePlayer("name", true));
+            AddParameter(new TypePlayer("name", true));
         }
 
         protected override void Execute(CallArgs args)
         {
-            Validate.IsTrue(args.Sender.HasValue, "This command can't be used by CONSOLE");
-
-            Player otherPlayer = args.Get<Player>(0);
-            args.Sender.Value.Teleport(otherPlayer.Position);
+            Player player = args.Get<Player>(0);
+            Player destination = args.Get<Player>(1);
 
-            SendMessage(args.Sender, $"Teleported to {otherPlayer.Name}");
+            player.Teleport(destination.Position);
+            SendMessage(player, $"Teleported to {destination.Name}");
```

Comment:
Maybe we can implement it so that you can warp yourself with one parameter and somebody else with two?